src/components/widgets-settings/source-selector/source-selector.tmpl.html

```diff
@@ -0,0 +1,10 @@
+<div class="settings source-selector">
+  <h5 translate>impac.widget.settings.source-selector.select_source</h5>
+
+  <div class="widget-lines-container">
+    <div class="widget-line" ng-repeat="app in organizationApps">
+      {{app.label}}
```

Comment:
Best to wrap this text in a more semantic element, like `<p>`, `<span>`, or `<label>` (https://www.w3.org/TR/html5/grouping-content.html#the-div-element)